libs/domain/picking/src/models/picking-list-qualifier.ts

```diff
@@ -8,6 +8,7 @@ export interface PickingListQualifier
     PageableQualifier,
     SearchQualifier {
   id?: string;
+  ids?: string[];
```

Comment:
Just replace `id` prop with `ids` we do not need both.

Reply:
We discussed with Taras that for now we should implement filtering by ids in the adapter and once BAPI supports it we will just update adapter code.